packages/core/src/types/oas3_1.ts

```diff
@@ -127,6 +127,8 @@ const Schema: NodeType = {
     dependentSchemas: listOf('Schema'),
     prefixItems: listOf('Schema'),
     contains: 'Schema',
+    minContains: { type: 'number' },
```

Comment:
Is there an integer type?

Reply:
@adamaltman looks like we used it as type in 2.0, but don't have it starting from 3.0. Maybe @RomanHotsiy has more context regarding this.

Reply:
We should use `integer` and fix 3.0 schema usage too.

Reply:
@RomanHotsiy I rechecked and looks like the integer type in 2.0 was used according to the spec. And in 3.0 and older there seem no fields with type integer, besides the `type` itself.

Example:
<img width="677" alt="image" src="https://user-images.githubusercontent.com/54616703/169335964-182a4dba-5dd8-43cf-a6ff-081927f12698.png">

Reply:
OAS3 and later Schema type is covered by JsonSchema. See here: https://json-schema.org/draft/2020-12/json-schema-validation.html#rfc.section.6.3.1